plugins/defaultplugins/common/model/srv6/keys_agent_srv6.go

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2018 Bell Canada, Pantheon Technologies and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package srv6
+
+import (
+	"net"
+
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+// Keys and prefixes(to keys) used for SRv6 in ETCD key-value store
+const (
+	basePrefix     = "vpp/config/v1/srv6/"
+	localSIDPrefix = basePrefix + "localsid/" // full key is in form .../localsid/{sid}
```

Comment:
Keys should only use some custom name to reference some specific key, keep rest of the data inside the model.

Reply:
fixed in ea181a4262e68973038f769dcf05f0c9cc682053